src/installer/pkg/sfx/installers/netstandard2.1.proj

```diff
@@ -0,0 +1,60 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <SkipBuild Condition="'$(RuntimeFlavor)' != '$(PrimaryRuntimeFlavor)' or '$(TargetsMobile)' == 'true'">true</SkipBuild>
+    <GenerateInstallers>true</GenerateInstallers>
+    <InstallerName>netstandard-targeting-pack-2.1</InstallerName>
+    <PackageBrandNameSuffix>NETStandard.Library.Ref</PackageBrandNameSuffix>
+    <ProductBrandName>NETStandard.Library.Ref 2.1.0</ProductBrandName>
+    <VersionInstallerName>false</VersionInstallerName>
+    <PlatformPackageType>ToolPack</PlatformPackageType>
```

Comment:
I think this is technically a ref pack, not a tool pack.
```suggestion
    <PlatformPackageType>TargetingPack</PlatformPackageType>
```

Reply:
I see this as a repackaging project, where we define most of the properties in the project, instead of relying on infra to deduce them based on package type.

`PlatformPackageType` property conditions various values in packaging infra. Making this a ref-pack would produce a different package and require other changes.

I don't think it would be worthwhile to make this, and various additional, changes, to massage this into a real ref-pack project.